docs/raw/applications/applications.md

```diff
@@ -9,7 +9,7 @@ oidc_applications
 
 - Type: `list` of `applications.OIDC` 
```

Comment:
Please exclude the whole file from the PR, I have a much more descriptive information waiting on a different PR.